src/_nebari/stages/kubernetes_ingress/__init__.py

```diff
@@ -224,6 +230,17 @@ def set_outputs(
 
         super().set_outputs(stage_outputs, outputs)
 
+    def post_deploy(self, stage_outputs: Dict[str, Dict[str, Any]]):
```

Comment:
This was added due to the auto-provisioning not working. We need to make sure this is preserved.